ml-agents/mlagents/trainers/trainer_util.py

```diff
@@ -1,6 +1,5 @@
 import os
-import yaml
-from typing import Any, Dict, TextIO
+from typing import Any, Dict
```

Comment:
The `Any` here refers to the trainer_config param to `TrainerFactory`.  Should that be updated with the new class name?

Reply:
👍